bridge/src/main/java/com/livefront/bridge/BridgeDelegate.java

```diff
@@ -62,6 +64,10 @@
         mSavedStateHandler = savedStateHandler;
         mViewSavedStateHandler = viewSavedStateHandler;
         registerForLifecycleEvents(context);
+        mDiskHandler = new FileDiskHandler(context, mExecutorService);
+
+        // Clear out any data from old storage mechanism
+        mSharedPreferences.edit().clear().apply();
```

Comment:
Is there any plan to remove this at some point?  Or are you just going to leave this here indefinitely?

Reply:
Yeah that's a good question. I'm not exactly sure the best way to go about getting rid of this. I would think that I'd have to keep it around for at least a few versions and then make a note when I finally remove it.